kaggle_environments/agent.py

```diff
@@ -105,10 +105,11 @@ def __init__(self, raw, configuration, environment):
         self.raw = raw
         self.agent = None
 
-    def act(self, observation, timeout=10):
```

Comment:
Is this timeout default intentionally removed?

Reply:
Yep! This timeout is actually just calculated from the configuration so we pull it from the config rather than passing it in now.